indexer/editable_map_object.cpp

```diff
@@ -94,12 +97,37 @@ void EditableMapObject::SetNearbyStreets(vector<string> && streets)
 {
   m_nearbyStreets = move(streets);
 }
-void EditableMapObject::SetHouseNumber(string const & houseNumber)
+
+bool EditableMapObject::SetHouseNumber(string const & houseNumber)
 {
-  // TODO(AlexZ): Check house number for validity with feature::IsHouseNumber ?
+  strings::UniString us = strings::MakeUniString(houseNumber);
+  strings::NormalizeDigits(us);
+  // TODO: Improve this basic limit.
+  auto constexpr kMaxHouseNumberLength = 15;
```

Comment:
Да, @Zverik посчитал.

> On Apr 6, 2016, at 21:46, ygorshenin <notifications@github.com> wrote:
> 
> In indexer/editable_map_object.cpp <https://github.com/mapsme/omim/pull/2802#discussion_r58760310>:
> 
> >  {
> > -  // TODO(AlexZ): Check house number for validity with feature::IsHouseNumber ?
> > +  strings::UniString us = strings::MakeUniString(houseNumber);
> > +  strings::NormalizeDigits(us);
> > +  // TODO: Improve this basic limit.
> > +  auto constexpr kMaxHouseNumberLength = 15;
> Ты собирал статистику по длинам строк с номерами домов?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2802/files/2d1ff55e0585d141ddfbe0d7474e5eafa30edc9c#r58760310>